src/site/stages/build/index.js

```diff
@@ -133,7 +132,6 @@ function defaultBuild(BUILD_OPTIONS) {
     'Add permalinks and change foo.md to foo/index.html',
   );
 
-  smith.use(createFeatureToggles(BUILD_OPTIONS), 'Create feature toggles');
```

Comment:
we don't use this anymore

Reply:
this was an attempt to bootstrap the flag values into the static content for faster loading. it doesn't meet our use case since our assets are only generated once a day. https://docs.launchdarkly.com/sdk/client-side/javascript#bootstrapping